Source/Controllers/DataControllers/SPDatabaseStructure.m

```diff
@@ -314,10 +314,7 @@ - (void)queryDbStructureWithUserInfo:(NSDictionary *)userInfo
 		 }
 
             NSString *theQuery = [NSString stringWithFormat:@"SHOW FULL COLUMNS FROM %@ FROM %@", [aTableName backtickQuotedString], [currentDatabase backtickQuotedString]];
-
-            CLS_LOG(@"theQuery: %@", theQuery);
             SPLog(@"theQuery: %@", theQuery);
-            [[FIRCrashlytics crashlytics] setCustomValue:mySQLConnection.serverVersionString forKey:@"serverVersion"];
```

Comment:
Can we do a similar thing with MSAppCenter? To set specific analytics params

Reply:
We can have attachments when pre-processing crashes, but I've not yet implemented them

Reply:
You can add [custom events](https://docs.microsoft.com/en-us/appcenter/sdk/analytics/ios):

```objc
[MSACAnalytics trackEvent:@"something clicked"];
```
or 
```objc
NSDictionary *properties = @{@"Category" : @"LoadData", @"TimeTaken" : _warning._time2};
[MSACAnalytics trackEvent:@"Init LoadData" withProperties: properties];
```

You can pause and resume, when network bandwidth is needed, e.g. exporting a big table"
```objc
[MSACAnalytics pause];
[MSACAnalytics resume];
```
But this is for analytics, not crash reports.